pom.xml

```diff
@@ -97,8 +97,9 @@
         <!-- Kafka and it's dependencies MUST reflect what the Kafka version uses -->
         <version.kafka>3.1.0</version.kafka>
         <version.zookeeper>3.6.3</version.zookeeper>
-        <version.jackson>2.12.3</version.jackson>
-        <version.jackson.databind>2.12.3</version.jackson.databind>
+        <!-- Kafka uses 2.12.3 but we need to bump due to CVE-2020-36518 before Kafka fixes it -->
+        <version.jackson>2.13.2</version.jackson>
```

Comment:
LGTM, any reason though for having two variables? Couldn't `version.jackson` be used for all the artifacts?

Reply:
@gunnarmorling It is for productization purposes. There were some versions we needed to decouple jackson and databind as the redhat suffix was different.

Reply:
Ok, I see. Worth a comment there probably then. Other than that, LGTM. Leaving it to you to merge, as I'm not sure what status main is in. Thanks!